build

```diff
@@ -377,6 +386,8 @@ if [ -z "$gitstatus_cpu" ]; then
     armv6l)         gitstatus_cpu=armv6;;
     armv7l)         gitstatus_cpu=armv7;;
     aarch64)        gitstatus_cpu=armv8-a;;
+    ppc64le)        gitstatus_cpu=native;;
```

Comment:
Is there anything more specific/restrictive? I try to go with smaller instruction sets to make it easier to move compiled binaries from one machine to another.

Reply:
Done! I think it's better to go with `powerpc64le`.